src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/role/entities/Assignment.java

```diff
@@ -0,0 +1,210 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.auth.role.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.google.common.base.Objects;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.role.entities.enums.ActorIdType;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.role.entities.enums.Classification;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.role.entities.enums.GrantType;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.role.entities.enums.RoleCategory;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.role.entities.enums.RoleType;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@SuppressWarnings("PMD.ExcessiveParameterList")
+public class Assignment {
+
+    private UUID id;
```

Comment:
Suggestion: I am aware the role-assignment team defines this as UUID. However, in our case, I don't see why we need to keep UUID and carry with it rather than just use a simple String.

Reply:
Thanks I'll apply suggestion